.werft/workspace-run-integration-tests.yaml

```diff
@@ -97,7 +97,7 @@ pod:
         werft log result "slack notification" "${PIPESTATUS[0]}"
 
         werft log phase "clean up" "clean up"
-        git push origin "${BRANCH}" | werft log slice "clean up"
+        git push origin :"${BRANCH}" | werft log slice "clean up"
```

Comment:
How about this?
```suggestion
        git push origin HEAD | werft log slice "clean up"
```

Reply:
This is to clean up the remote temporary branch after the integration test job is finished, previously I missed adding ":" behind it.